ui/shared/components/panel/variants/Annotations.jsx

```diff
@@ -265,12 +265,21 @@ const Annotations = React.memo(({ variant, mainGeneId, showMainGene }) => {
           {svSizeDisplay(end - pos)}
         </b>
       )}
-      {algorithms && (
+      {(algorithms || bothsidesSupport) && (
         <b>
           <HorizontalSpacer width={5} />
           <Popup
             trigger={<Icon name="help circle" />}
-            content={`Algorithms: ${algorithms}`}
+            content={
+              <div>
+                {algorithms && `Algorithms: ${algorithms}.`}
+                {bothsidesSupport && (
+                  <div>
+                    Variant has read-level support for both sides of breakpoint
```

Comment:
per my comment on the ticket, this should just be "Bothsides Support"

Reply:
Update pushed.